### ISSUE TITLE
Keyboard shortcut typo for "Select Next Layer" and "Select Previous Layer"

### ISSUE THREAD

**Issue by MarkOates** [mapeditor/tiled#3885]
The keyboard shortcuts in the `Layer` menu have "Select Previous Layer" and "Select Next Layer" listed as `Command`+`UP` or `Command`+`DOWN`.  However, the actual working keyboard shortcut is `Option`+`UP` and `Option`+`DOWN`.

![level_1-0x tmj (some_platformer-01) 2024-02-05 17-45-57](https://github.com/mapeditor/tiled/assets/772949/be760f4a-5354-4875-91f1-cd95e0cb0b6f)

**Specifications:**
- OS: MacOS Sonoma 14.2.1 (23C71)
- Tiled Version: 1.10.2

**Comment by bjorn** [comment]
Hmm, these shortcuts Ctrl + PageUp / Ctrl + PageDown by default, which should translate to Cmd + PageUp / Cmd + PageDown on macOS. Maybe the arrow icons are just a bit confusing? You can see the arrows used for Up / Down in the Raise / Lower Layers actions.

**Comment by MarkOates** [comment]
I see! That is it.  The `PageUp`/`PageDown` icons are what confused me.

I did not know this, but, it looks like Apple does have symbols specifically for `PageUp`/`PageDown`, different from the `Up`/`Down` arrows: https://support.apple.com/en-ca/guide/mac-help/cpmh0011/mac

<img width="300" alt="What are those symbols shown in menus on Mac? - Apple Support (CA) 2024-02-06 10-59-28" src="https://github.com/mapeditor/tiled/assets/772949/33bfaf0d-5d98-42d0-a2cf-e4624af85b09">

And, the standard arrows are the line symbols, not the triangles.  Which is funny, because the symbols on my keyboard are triangles.

![keyboard-keys](https://github.com/mapeditor/tiled/assets/772949/007c90c0-07f5-4672-b4fc-5168130fa686)

**Comment by bjorn** [comment]
@MarkOates Right, I have no idea why it uses those other arrows for Up / Down and appears to use the Up / Down arrows for PageUp / PageDown. I wonder if it might be an issue in Qt. On the other hand, this is a native macOS menu, and it does highlight the menu when those keys are pressed, so it might also be an inconsistency in macOS. :man_shrugging:

**Comment by MarkOates** [comment]
OK, I did a quick skim online and across a few different applications on my Mac and I wasn't able to find any active Qt issues or mentions of it, or even another application on my Mac that had the PageUp or PageDown icon to cross-reference.

Enough cost/payoff digging for me! I'm happy to close the issue if you like. Thanks. :)

**Comment by bjorn** [comment]
@MarkOates Thanks for doing some digging! I also had a quick look but could not find anything either. It does seem like a valid issue, so I'll keep it open for later.
